babel.config.js

```diff
@@ -1,58 +1,48 @@
-module.exports = function (api) {
-  const validEnv = ['development', 'test', 'production'];
-  const currentEnv = api.env();
-  const isDevelopmentEnv = api.env('development');
-  const isProductionEnv = api.env('production');
-  const isTestEnv = api.env('test');
+// See also Ruby config/webpack/environment.js
+//
+// See docs at https://babeljs.io/docs/en/options
+//
+// Initialized based on default config from webpacker at
+// https://github.com/rails/webpacker/blob/master/package/babel/preset.js
```

Comment:
Most of these changes were just pulled automatically from the preset documented here, aside from our app-specific plugins